doc/user/install_guide.md

```diff
@@ -38,7 +38,7 @@ kubectl delete clusterrolebinding etcd-operator
 ```
 
 ## Installation via Helm
-**Disclaimer:** The following Helm chart is an external project not maintained by the etcd-operator maintainers. It may not be up to date and so is not the recommended way to install the etcd-operator.
+**Disclaimer:** The following Helm chart is an external project not maintained by the etcd-operator maintainers, so it may not be up to date.
```

Comment:
prefer `;` to connect two closely full sentence together.
`maintainers; so it` or `maintainers; the chart` 

Just a suggestion.